frontend/public/components/_resource.scss

```diff
@@ -95,3 +95,7 @@ $height: 18px;
 .co-m-resource-icon--align-left {
   margin-left: 0;
 }
+
```

Comment:
I don't see this class used anywhere.

Reply:
`.co-m-resource-icon--align-left` is used on the ingress view.